schema/me/conversation/index.js

```diff
@@ -190,7 +190,28 @@ export const ConversationFields = {
   last_message_open: {
     type: GraphQLString,
     description: "Timestamp if the user opened the last message, null in all other cases",
-    resolve: conversation => null, // eslint-disable-line no-unused-vars
+    resolve: (conversation, options, request, { rootValue: { accessToken } }) => {
+      const radiationMessageId = get(conversation, "_embedded.last_message.radiation_message_id")
+      const impulseParams = {
+        conversation_id: conversation.id,
+        radiation_message_id: radiationMessageId,
+        "expand[]": "deliveries",
+      }
+      return gravity.with(accessToken, { method: "POST" })("me/token", {
+        client_application_id: IMPULSE_APPLICATION_ID,
+      }).then(data => {
+        return impulse.with(data.token, { method: "GET" })(
+          `message_details`,
+          impulseParams
+        ).then(({ message_details }) => {
+          if (message_details.length === 0) {
+            return null
+          }
+          const relevantDelivery = message_details[0].deliveries.find(d => d.email === conversation.from_email)
```

Comment:
Unfortunately yes. We need to request a token from grav in order to fetch from impulse. 

The next iteration of optimizing can be to fetch all the messages via id from impulse, to batch load those.

But currently yea, this is the pattern we've been using so it is in line with that. Can be improved upon.